layouts/shortcodes/support.html

```diff
@@ -6,6 +6,7 @@
     <a href="{{ .url }}" target="_blank">
       <img src="{{ $logo }}">
     </a>
+    <p class="is-centered">{{ .name }}</p>
```

Comment:
@thisisobate I suck at styling, would you mind helping me better format the name? If it's too annoying for you, we can do it ina separate PR as well.

Reply:
@hisisobate Would you have some time to help on the styling here?

Reply:
Hi...sorry for the late response. For some weird reason, I couldn't get notified by GitHub about your comment.

Would take a look now and see how best to do te styling.